inst/rmarkdown/templates/cholera_outbreak/skeleton/skeleton.Rmd

```diff
@@ -1390,24 +1388,42 @@ The following will plot a map based on the attack rate table.
 ```{r choropleth_maps, message = FALSE, warning = FALSE}
 
 ## Checklist for plotting ----------------------------------------------
+## - [ ] consider making categorical groupings of your AR or counts variable
 ## - [ ] merge your attack rate or counts table with your shapefile 
 ## - [ ] Choose the appropriate section below depending on if you are plotting polygons or points
 ## - [ ] Choose what variable you would like to fill (for polygons) or colour (for points) with (counts or AR)
 ## - [ ] DELETE OR COMMENT OUT THE SECTION YOU WONT BE USING
 
-# join your ar or case counts based on matching shapefile names with regions in your ar table
+## create a categorical variable for plotting 
```

Comment:
I think making a categorical variable is not the way to go here... it would be better just to manipulate the breaks/labels in ggplot2 as opposed to manipulating the data itself.

Reply:
Not convinced either. If you can think of a better way to get the zero group and then label the others the same way as this does then have at it.

Reply:
One way of doing it would be to set the lower limit to 1 so that zero is effectively NA

Reply:
When you do that, then all the zero counts are greyed out, but that's effectively what you want, right? A way to differentiate zero from not-zero?

Reply:
Guess that could work. Though I guess I  certain situations NA would be for places without surveillance

Reply:
> Guess that could work. Though I guess I certain situations NA would be for places without surveillance

That's a good and valid point!

Reply:
Also, this may be me being naive, but why is it necessary to highlight zero specifically? What would it mean if there is one area that has zero cases vs an area that has a single or even five cases?

Reply:
Because zero is zero and one is one :) 
To dentify not affected areas... Or potentially to highlight that your system is not working (more likely!)

Reply:
> Because zero is zero and one is one :)
> To dentify not affected areas... Or potentially to highlight that your system is not working (more likely!)

But that assumes complete and accurate coverage. A zero count does not necessarily mean that there are zero cases, it just means that zero cases were counted by whatever survey mechanism was present.

Reply:
Exactly why its important to know. Rather than not knowing.... Which tells you nothing

Reply:
You've got a point. While a count of one or two indicates lower prevalence, a zero count could either indicate low prevalence OR that something is wrong with the survey.

Reply:
So, I've added a new function called `find_breaks()` which will calculate breaks at evenly-spaced intervals. It now produces a plot that looks like this:

![image](https://user-images.githubusercontent.com/3639446/53680520-db361a00-3cd4-11e9-90f3-77b02f07eafa.png)

Right now it doesn't include the highest value, but that can be achieved by adding `ceiling = TRUE`.